requirements-testing.txt

```diff
@@ -6,3 +6,4 @@ ndx-miniscope
 spikeinterface[qualitymetrics]>=0.100.0
 zarr<2.18.0  # Error with Blosc (read-only during decode) in numcodecs on May 7; check later if resolved
 pytest-xdist
+ndx-pose
```

Comment:
Alas, we can not have this here - a fundamental part of the early testing suite (in CI) is to mimic minimal installation patterns, which will necessarily cause an import error for the optional/extra installation of DLC dependencies